src/wazuh_db/wdb_parser.c

```diff
@@ -5558,6 +5566,38 @@ int wdb_parse_global_sync_agent_info_set(wdb_t * wdb, char * input, char * outpu
     return OS_SUCCESS;
 }
 
+int wdb_parse_get_groups_integrity(wdb_t* wdb, char* input, char* output) {
+    cJSON *response = NULL;
+    char *out = NULL;
+    char *hash = input;
+    int ret = OS_SUCCESS;
+
+    response = cJSON_CreateArray();
+    int groups_synced = wdb_global_groups_synced(wdb);
+
+    if (groups_synced == 1) {
```

Comment:
Avoid the usage of raw numbers without a definition.

Reply:
It's confusing, you're right